resources/public/api/conf/3.0/schemas/broughtForwardLosses/listBFLoss/list_request.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
These fields are the query parameters not the request. They have duplicated on the RAML preview under the responses

Reply:
it only has a response and that is should only show the response body in the schemas